sql/driver/wire.proto

```diff
@@ -71,6 +71,11 @@ message Response {
     // presence indicates that the operation was a DDL statement.
     message DDL {}
 
+    // Ack is a message containing the tag of a result without a meaningful return.
+    message Ack {
+      required string tag = 1 [(gogoproto.nullable) = false];
```

Comment:
I think we never use `required` (even if most of our fields aren't optional), see https://developers.google.com/protocol-buffers/docs/proto#required_warning.